Data-Structures/Linked-List/test/SinglyLinkedList.test.js

```diff
@@ -247,4 +247,10 @@ describe('SinglyLinkedList', () => {
     headNode.rotateListRight(5)
     expect(headNode.get()).toEqual([20, 30, 40, 50, 10])
   })
+
+  it('Reverse a number represented as a Linked List where each node has a single integer', () => {
+    const headNode = new LinkedList([4, 3, 1])
```

Comment:
This is not a headNode but rather a list object

Reply:
Changed it to list.